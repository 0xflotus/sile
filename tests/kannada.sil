\begin[papersize=a4]{document}
Hello SILE! \font[family=Noto Sans Kannada,size=10pt,language=kn]{ನಮಸ್ಕಾರ :)}
\end{document}
