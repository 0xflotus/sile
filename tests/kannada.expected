<tests/kannada.sil>
Set paper size 	419.5275636	595.275597
Begin page
Mx 	40.97638
My 	37.84972
Set font 	Gentium;10;200;normal;normal;;LTR
T	40 69 76 76 79	(Hello)
Mx 	65.25424
T	51 41 44 37 209	(SILE!)
Mx 	88.10633
Set font 	Noto Sans Kannada;10;200;normal;normal;;LTR
T	39 44 163 57 97 46	(ನಮಸ್ಕಾರ)
Mx 	130.10695
T	422	(:)
Mx 	132.78761
T	405	())
[1] Mx 	207.41759
My 	553.73265
Set font 	Gentium;10;200;normal;normal;;LTR
T	17	(1)
End page
Finish

